types/youtube-dl/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "files": [
+        "index.d.ts",
+        "youtube-dl-tests.tsx"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
```

Comment:
This doesn't seem to actually require "dom", except for the `console.log` calls (see below comment).